events-amd.js

```diff
@@ -9,7 +9,7 @@ define('event', [], function() {
 
 	function removeEvent(el, eventName, named) {
 		if (el['_event']) {
-			if (named) {
+			if (named && el['_event'][eventName]) {
```

Comment:
This verification should be inside this If, cause If it's the second condition returns false, it'll remove all events of given `eventName`.